nomos-da/kzgrs-backend/src/common/blob.rs

```diff
@@ -72,3 +72,118 @@ impl blob::Blob for DaBlob {
         self.column_idx.to_be_bytes()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::common::blob::DaBlob;
+    use crate::common::{deserialize_canonical, serialize_canonical};
+    use crate::common::{Chunk, Column};
+    use crate::encoder::{DaEncoder, DaEncoderParams};
+    use crate::global::GLOBAL_PARAMETERS;
+    use kzgrs::{Commitment, Proof};
+    use nomos_core::da::DaEncoder as _;
+    use rand::{thread_rng, RngCore};
+    use serde::{Deserialize, Serialize};
+
+    pub fn get_da_blob(data: Vec<u8>) -> DaBlob {
+        let encoder_params = DaEncoderParams::new(2048, true, GLOBAL_PARAMETERS.clone());
+        let encoder = DaEncoder::new(encoder_params);
+        let encoded_data = encoder.encode(&data).unwrap();
+        let columns: Vec<_> = encoded_data.extended_data.columns().collect();
+
+        let index = 0;
+        let da_blob = DaBlob {
+            column: columns[index].clone(),
+            column_idx: index
+                .try_into()
+                .expect("Column index shouldn't overflow the target type"),
+            column_commitment: encoded_data.column_commitments[index],
+            aggregated_column_commitment: encoded_data.aggregated_column_commitment,
+            aggregated_column_proof: encoded_data.aggregated_column_proofs[index],
+            rows_commitments: encoded_data.row_commitments.clone(),
+            rows_proofs: encoded_data
+                .rows_proofs
+                .iter()
+                .map(|proofs| proofs.get(index).cloned().unwrap())
+                .collect(),
+        };
+
+        da_blob
+    }
```

Comment:
I believe this already supplied under the new `testutils` feature in `kzgrs_backend`, in case you want to use it.

Reply:
Yes it was somehow provided, but not flexible enough as the encoded settings was prefixed to 16 columns and I needed to tweak it further.